docs/candidate-app/candidates.md

```diff
@@ -26,11 +26,10 @@ You can create a new user using the "Create new entry" button. The following fie
 Then, use the "Save" button and you should be able to log in as the candidate at http://localhost:5173/candidate.
 
 ## Resetting Password
-The user gets an email with a link to reset their password using the forgot password functionality on the login page. The frontend URL in the emails is configured in `.env` with the PUBLIC_FRONTEND_URL variable, and the SMTP can be configured using the following variables:
-- `SMTP_HOST`: the hostname the SMTP server runs on
-- `SMTP_PORT`: the port of the SMTP server
-- `SMTP_USERNAME`: the username used for authentication
-- `SMTP_PASSWORD`: the password used for authentication
+The user gets an email with a link to reset their password using the forgot password functionality on the login page. The frontend URL in the emails is configured in `.env` with the PUBLIC_FRONTEND_URL variable, and the email service (AWS SES) can be configured using the following variables:
+- `AWS_ACCESS_KEY_ID`: AWS SES user access key
+- `AWS_SECRET_ACCESS_KEY`: AWS SES user secret access key
+- `AWS_REGION`: AWS SES region
 - `MAIL_FROM`: the email address the emails are sent from
```

Comment:
We should add a note here (and in `.env.example`) that you need also to set these addresses in Strapi > Settings > Email Templates for all templates, otherwise the functions associated with the templates will fail. Of course, if you can think of an easy way to insert these from `env` that'd be nice, but probably not worth the effort.